src/app/public/modules/overlay/overlay-config.ts

```diff
@@ -0,0 +1,32 @@
+import {
+  StaticProvider
+} from '@angular/core';
+
+export interface SkyOverlayConfig {
+
+  /**
+   * Specifies if the overlay should be closed after a navigation change.
```

Comment:
Probably not for an interface (since interfaces by definition do not have default values). The service's public method (that uses the config) would be a better place to document defaults.

Reply:
Default values should be handled by the implementing class. I'll add those to the service method below.